sound/lame/Makefile

```diff
@@ -56,6 +62,11 @@ ifeq ($(ARCH),i386)
 TARGET_CFLAGS+=-msse
 endif
 
+ifeq ($(CONFIG_LAME-LIB_OPTIMIZE_SPEED),y)
+	TARGET_CFLAGS += $(TARGET_CFLAGS) -O3 -fomit-frame-pointer -ffast-math -fschedule-insns2
```

Comment:
If you're doing `-O3` then `-fschedule-insns2` and `-fomit-frame-pointer` should already be on, according to the `gcc` man page.

And you're okay with the implications of `-ffast-math`?

Reply:
Thanks, I'll do a fixup...
Regarding -ffast-math it's fine in this case

ICC uses it by default and given that there hasn't been any noise over at hydrogenaudio.io and rarewares.org for years I'm going to assume that it's safe.